.github/workflows/test-python.yml

```diff
@@ -67,6 +67,7 @@ jobs:
       - name: Run tests
         run: >
           python3 django_repo/tests/runtests.py --settings mongodb_settings -v 2
+          aggregation
```

Comment:
Please add `aggregation_regress` as well.

Reply:
Yes, could be, or maybe we should work on it in another ticket, this one is too long I think.

There are some test that does not pass, some of them came from ordering issue, others from known issues as "QuerySet in annotate() is not supported on MongoDB" but others are simple errors.